lib/ansible/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -64,6 +64,14 @@
       - Possible values are true and false.
     type: bool
     default: yes
+  merge_method:
+    description:
+      - What requirements are placed upon merges.
+      - Possible values are merge, rebase_merge (Merge commit with semi-linear history), ff (fast-forward merges only)
```

Comment:
```suggestion
      - Possible values are C(merge), C(rebase_merge) (Merge commit with semi-linear history), C(ff) (fast-forward merges only)
```

Reply:
This doesn't seem "fixed" to me, can you try using `C()` as @Akasurde suggested?

Reply:
Hm. I'm sorry I thought I fixed that. I will look/fix later today. Thanks for pointing that out.

Reply:
Ah, I see what I did; I clobbered these changes with my rebase. Will re-fix and push up again. Sorry about that.